tests/unit/infrastructure/s3/s3Service.test.ts

```diff
@@ -0,0 +1,108 @@
+import { S3 } from 'aws-sdk';
+import getFromS3 from '../../../../src/infrastructure/s3/s3Service';
+
+describe('S3 Service', () => {
```

Comment:
I think we should be clearing and reseting the mocks as we are not sure when they will be used and it is also a good practice to start tests from a clean slate

`beforeEach()` & `afterEach`

We may be mutating the object here